src/gitlab-ci.coffee

```diff
@@ -132,6 +135,47 @@ module.exports = (robot) ->
             return
 
         return "#{major}.#{minor}.#{dev}"
+    triggerBuildPost = (res, updateRichmond, quickly, branch, buildTagPrefix) ->
+        triggerURL = 'https://agra.sdelements.com/api/v4/projects/33/trigger/pipeline/'
```

Comment:
As a follow-up, I also understand sometimes you just want to get something working and you hard-code references. Just at some point we have to make the decision to make the code production-ready, in which case we would turn these references into configuration.